common/test/acceptance/pages/lms/discussion.py

```diff
@@ -567,6 +567,12 @@ def click_element(self, selector):
         )
         self._find_within(selector).click()
 
+    def new_post_button_visible(self):
```

Comment:
I see a property in the base class is already available for checking new button.
[    def new_post_button(self):  ... ](https://github.com/edx/edx-platform/blob/master/common/test/acceptance/pages/lms/discussion.py#L29-L33)